morphocut/str.py

```diff
@@ -8,7 +8,31 @@
 @ReturnOutputs
 @Output("string")
 class Format(Node):
-    """Format a string just like :py:meth:`str.format`."""
+    """
+    Format a string just like :py:meth:`str.format`.
+    
+    Args:
+        fmt (str): A format in which we want our string to be.
+        *args: Arguments to be replaced with placeholders in fmt
+        _args: Arguments to be appended after *args
+        _kwargs: Key value paired arguments
+        **kwargs: Key value paired arguments to be appended after _kwargs
+
+    As positional arguments, :py:meth:`str.format` receives ``args`` then ``_args``.
+    As keyword arguments, :py:meth:`str.format` receives ``_kwargs`` then ``kwargs``.
+    This means that keys passed as keyword arguments overwrite keys in a dict passed as ``_kwargs``.
+
+    Example:
+        .. code-block:: python
+
+            with Pipeline() as pipeline:
+                result = Format(fmt, *args, _args=_args, _kwargs=_kwargs, **kwargs)
```

Comment:
Make this more concrete: Introduce stream variables, regular variables and a concrete `fmt`.